Build.ps1

```diff
@@ -1,85 +1,15 @@
-param(
-    [String] $majorMinor = "0.0",  # 2.0
-    [String] $patch = "0",         # $env:APPVEYOR_BUILD_VERSION
-    [String] $customLogger = "",   # C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
-    [Switch] $notouch,
-    [String] $sln                  # e.g serilog-sink-name
-)
+Push-Location $PSScriptRoot
```

Comment:
Thank you @nblumhardt for reviewing this.

I see there are [test sections](https://github.com/serilog/serilog/blob/dev/Build.ps1#L29-L49) which don't apply to this repro.  I could either add a check for the existence of `test` folder, or remote the sections for this repo.  Which approach do you prefer?